src/components/Skeletons/ItemListSkeletonView.tsx

```diff
@@ -20,20 +23,21 @@ function ItemListSkeletonView({shouldAnimate = true, renderSkeletonItem, fixedNu
         const items = [];
         for (let i = 0; i < numItems; i++) {
             items.push(
-                <SkeletonViewContentLoader
-                    key={`skeletonViewItems${i}`}
-                    animate={shouldAnimate}
-                    height={CONST.LHN_SKELETON_VIEW_ITEM_HEIGHT}
-                    backgroundColor={theme.skeletonLHNIn}
-                    foregroundColor={theme.skeletonLHNOut}
-                    style={themeStyles.mr5}
-                >
-                    {renderSkeletonItem({itemIndex: i})}
-                </SkeletonViewContentLoader>,
+                <View style={[themeStyles.mr5, itemViewStyle]}>
```

Comment:
Please move the `key` prop from `SkeletonViewContentLoader` to the `View` component since this the view is the current top level item in the list.

![Screenshot 2024-06-22 at 01 15 13](https://github.com/Expensify/App/assets/4160319/6460534c-705e-4c09-8c16-d04bfe488ec9)